src/adventures-photography/_posts/2017-08-09-eurovelo-albania.md

```diff
@@ -0,0 +1,148 @@
+
+In front of us the ground opened up as the road swung around to the left as the forest trees came to an abrupt stop, contouring along the vast valley which just appeared below us with near-vertical walls. We both pulled over to take in what was in front of us, and the endless line of spaced-out expedition vehicles along the ribbon of switch-backing tarmac down one side, over the bridge, and back up the other side. We both knew the descent was going to be a good one. 
+
+{% figure %}
+    {% img src: 2016-06-web-DSCF7426, alt: Descending into the first valley during our last day in Albania. %}
+    {% figcaption %}Descending into the first valley during our last day in Albania.{% endfigcaption %}
+{% endfigure %}
+
+Max let me go ahead so I could setup a couple of photographs and set off once I gave him a wave. It was hard to take a bad shot here. I quickly packed down and we both hit the road hard for the bridge, before grinding to a halt on the hard and steep climb out the other side. It was hard going — even some of the huge expedition 4x4s the rally teams were using were struggling on this climb with steep and tight corners.
+
+{% figure %}
+    {% img src: 2016-06-web-DSCF7431, alt: Crossing a bridge at the bottom of a valley. Look for Max through the gap in the trees for a sense of scale. %}
+    {% figcaption %}Crossing a bridge at the bottom of a valley. Look for Max through the gap in the trees for a sense of scale.{% endfigcaption %}
+{% endfigure %}
+
+Eventually we hit the plateau top again and spent a while cruising along a flat road in desolate terrain — sand-coloured bedrock as far as I could see with a black ribbon of tarmac straight down the middle, slowly winding its way along the latest line. It was on this road Max and I were surprised to see the Google Street-view car driving the other way — providing [the only picture of both of us as the same time](https://www.google.com/maps/@41.9809719,19.767932,3a,65.4y,107.4h,94.89t/data=!3m6!1e1!3m4!1sKxid__TWVUXI2AUX08NlBQ!2e0!7i13312!8i6656). 
+
+{% figure %}
+    {% img src: 2016-06-web-DSCF7445, alt: Climbing out of a valley on the second day. You can jsut see a National Rally truck peaking round the corner as it labours up the steep hill. %}
+    {% figcaption %}Climbing out of a valley on the second day. You can jsut see a National Rally truck peaking round the corner as it labours up the steep hill.{% endfigcaption %}
+{% endfigure %}
+
+Eventually we rounded a corner and saw the ground drop away again. Unlike the previous time though the ground didn’t rear up steeply on the other side — quite the country. This time it dropped as far as the eye could see, and we both got rather excited and the prospect of the decent down to the plains of Albania below. 
+
+{% figure %}
+    {% img src: 2017-08-09-google-maps-albania, alt: Our Google Street View portrait! %}
+    {% figcaption %}Our Google Street View portrait!{% endfigcaption %}
+{% endfigure %}
```

Comment:
Replaced the source image for this one, and disabled the watermark as it's not really a necessity.